journal.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
+Session = sessionmaker()
+DATABASE_URL = os.environ.get(
+    'DATABASE_URL',
+    'postgresql://jimgrant@localhost:5432/learning-journal'
+)
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    title = sa.Column(sa.String(length=128))
+    text = sa.Column(sa.Text)
```

Comment:
for the reasons discussed in class today, you'll probably want to change these two data types to be `Unicode` and `UnicodeText`.